Dockerfile-nightly.model

```diff
@@ -1,12 +1,12 @@
 FROM google/cloud-sdk:latest as nightly-storage
 
-RUN gsutil cp `gsutil ls gs://prestashop-core-nightly/ | grep -E 'develop.+\.zip$' | tail -1` /tmp/prestashop.zip
+RUN gsutil cp `gsutil ls gs://prestashop-core-nightly/ | grep -E 'develop.+\.zip$$' | tail -1` /tmp/prestashop.zip
```

Comment:
Why this double dollar sign?

Reply:
Because it's a template and $ is for variables.